JSON_for_IO-Link.yaml

```diff
@@ -4896,10 +4924,8 @@ components:
                   enum:
                     - '1.0'
                     - '1.1'
-                parameterChecksum:
+                parameterChecks:
```

Comment:
Should be changed back to `parameterChecksum`.

Reply:
Agree!